plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -429,4 +429,8 @@ module.exports = class ServiceTree extends Tree {
       return serviceTreeLabels;
     }, []);
   }
+
+  isFolder() {
```

Comment:
One can use the service instance type to differentiate between a service tree (folder) or an app, pod and framework. Let's please not add a method like this one.

Reply:
Correct, now do we want as a method in structs or inside `ServicesTable.js`? I see being part of the struct since this can be re-used by other parts of the application that has service, also because to keep consistent with what we already have.

Reply:
If you really need the method for `convenience` please consider adding a util that checks the instance type.

Reply:
I suggested the opposite in #2459 💃

Reply:
@nLight different case as this one refers to the service type and your comment was related to service state.